frontend/test/metabase/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -10,50 +10,73 @@ describe("scenarios > embedding > full app", () => {
     cy.intercept("GET", "/api/automagic-dashboards/**").as("getXrayDashboard");
   });
 
-  describe("navigation", () => {
+  describe("home page navigation", () => {
```

Comment:
Leave a few new happy cases and non-unit testable cases here in E2E.

Reply:
I believe that only 1 e2e test is needed that verifies that options passed via URL actually got accepted by the app bar. The rest can be tested with integration tests in Jests. cc @nemanjaglumac WDYT?